tools/distpartitioning/convert_partition.py

```diff
@@ -175,109 +175,61 @@ def _get_unique_invidx(srcids, dstids, nids):
     return uniques, idxes, srcids, dstids
```

Comment:
[Optional] Since you changed almost 80% of the file, I'd suggest you to also update the other piece to follow the convention.